src-java/floodlight-service/floodlight-modules/src/test/java/org/openkilda/floodlight/command/flow/ingress/IngressFlowSegmentCommandJsonTest.java

```diff
@@ -56,7 +56,7 @@ protected IngressFlowSegmentRequest makeRequest() {
                 new SwitchId(20),
                 8,
                 new FlowTransitEncapsulation(9, FlowEncapsulationType.TRANSIT_VLAN),
-                new RemoveSharedRulesContext(false, false));
+                new RemoveSharedRulesContext(false, false, false));
```

Comment:
nit: maybe switch to builder? or create final constants of remove shared rules context instance

Reply:
fixed